indico/core/celery/templates/celery/celery_tasks.html

```diff
@@ -0,0 +1,55 @@
+{% from 'message_box.html' import message_box %}
+{% extends 'layout/base.html' %}
+
+{% block title %}
+    {% trans %}Tasks{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div class="tasks">
+        {% call message_box('info', fixed_width=True) %}
+            {% if flower_url %}
+            {% trans %}Monitor Indico related tasks with flower <a class="flower-url" href="{{ flower_url }}" target="_blank">here</a>.{% endtrans %}
```

Comment:
`{% trans -%}...{%- endtrans %}` to avoid leading/trailing whitespace in the extracted i18n data